### PR TITLE
Add contributors routes

### PR DESCRIPTION
**Pull request by grmbyrn**
* Add a link on the `How to Contribute` section of the home page to link to a `contributors` route. Each contributor's image and name links to an individual contributor page with information on that person.
* Add link to each blog post of the person who wrote that post, which links to that person's contributor page.
* Each `contributor` route imports `ContributorCard` from `molecules` in `components`. Info about that contributor is passed to `ContributorCard` component to avoid repetition of code and styles.
* Add `contributor` and `contributorSlug` to `type BlogPost` to avoid TypeScript errors.

### CODE REVIEW

File: src/routes/contributors/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	.contributor-biography {
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+		text-align: center;
+		max-width: 700px;
+		margin: 0 auto;
+		margin-top: 2rem;
+		line-height: 1.5;
+	}
```
Comment (josecelano) [inline src/routes/contributors/+layout.svelte]: Hi @grmbyrn why do you need this also here? It's also defined [inside the component](https://github.com/torrust/torrust-website/pull/108/files#diff-26602b02881602b43ba9d57dded530229319993290ded5addaf9ef90fa5b653eR49-R59).

**Comment by josecelano** [comment]
@grmbyrn I think the link in the blog post to the contributor is also missing.

**Review by josecelano — CHANGES_REQUESTED** [review]
Hi @grmbyrn It looks good!, but:

- There is still duplicated CSS for the class `.contributor-biography`
- The link to the contributor is missing gin the article.

File: src/routes/contributors/+layout.svelte (hunk 1 above)
Comment (grmbyrn) [reply src/routes/contributors/+layout.svelte]: Hi @josecelano. I added the contributor link to each article and removed duplicated css.

**Comment by da2ce7** [comment]
I think that we should include some sort of section for the contributors.

`.../contributors/name/`

Then we could have:

`.../contributors/`

To be an index of the contributors?

What do you think?

**Pull request closed without merging**